components/DropDown/BloodBox.js

```diff
@@ -17,7 +17,7 @@ function BloodBox({ data, setData }) {
   return (
     <>
       {data.map((el, i) => (
-        <Flex key={i} justifyContent="left" marginLeft="1.5" marginY="1.5">
```

Comment:
Hey @0marwa0 Nice Work, But there was no need to delete those props because they were for the dropdown

Reply:
@AhmadSaman I deleted bc it doesn't work in Arabic alignment
![dropdown](https://user-images.githubusercontent.com/14044653/140308082-dc87c27a-1cc0-4f2f-9265-18e6e5873d63.png)